src/domain/enrolment/__tests__/CreateEnrolmentPage.test.tsx

```diff
@@ -174,17 +136,64 @@ jest.setTimeout(20000);
 // "svg elements with an img role have an alternative text (svg-img-alt)"
 test.skip('page is accessible', async () => {
   const { container } = render(<CreateEnrolmentPage />, {
-    mocks: mock1,
+    mocks: PageMockEnrolTimeNotStarted,
     query: { eventId: eventId, occurrences: occurrenceIds },
   });
 
   await act(wait);
   expect(await axe(container)).toHaveNoViolations();
 });
 
+test('has enrolment title and button', async () => {
+  render(<CreateEnrolmentPage />, {
+    mocks: PageMockWithSelectedPlace,
+    query: { eventId: eventId, occurrences: occurrenceIds },
+  });
+
+  await act(wait);
```

Comment:
If you have time, you could try get rid of these act(wait) statements. Is there loading spinner or something that could be waited? or:

```
await waitFor(() => {
expect(
    screen.getByRole('heading', { name: 'Ilmoittaudu tapahtumaan' })
  ).toBeInTheDocument();
})

```

Reply:
(pretty minor problem, but it is hacky workaround)

Reply:
I think that checking `toBeInTheDocument` is redundant, it's easier to put something like:
`await screen.findByRole('heading', { name: 'Ilmoittaudu tapahtumaan' });`
No need for acts either.

Reply:
yes in this case at least findBy is better